data/field/field.urdf

```diff
@@ -9,6 +15,13 @@
             <geometry>
                 <mesh filename="field.obj" scale="1 1 1"/>
             </geometry>
+   
+<friction>      
+        <ode>      
```

Comment:
I've been unable to find where these `<friction>` and `<ode>` tags come from. Could you link me a reference.

Reply:
I tried the <friction> and <ode> to the Field.urdf but it didn't work so I found the friction from a Japanese website. Here is the link :https://qiita.com/saketosakana/items/0e52997cbe43919ee2bd 
https://qiita.com/saketosakana/items/491d3e0cec0a0e288748
https://qiita.com/saketosakana/items/b4529df7ebc1f60356a6